lib/game.rb

```diff
@@ -0,0 +1,128 @@
+class Game
+    attr_accessor :inputsX, :inputsO, :play_again, :valid_input, :active_player
+    def initialize(player1, player2)
+      @board = Board.new  
```

Comment:
Game shouldn't be initializing other Classes (board and player classes). What you should have done is pass board, player1 and player2 as initialized objects already. You are indeed passing `player1` and `player2` as parameters here but they are simply the "X" and "O" strings. 

player1 = Player.new("X"), player2 = Player.new("O") and board = Board.new should have been run outside of the Game class. The initialized player1, player2 and board objects should have been arguments in initializing Game class.